src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
+@import "./variables";
+@import "./stars.scss";
+
 body {
   margin: 0;
+  font-family: "Roboto", "Open Sans", sans-serif;
+  color: #060b35;
+
+  h1,
+  h2,
+  p {
+    margin: 0;
+  }
+
+  .product-card {
+    width: 200px;
+    padding: 32px 16px 16px;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    box-sizing: border-box;
+  }
+
+  .product-card__image {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  .product-card__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  .product-card__product-code {
+    margin-top: 4px;
+    color: #616070;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  .reviews {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &__stars {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+  }
+
+  .reviews__rating {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  .price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  .description {
+    color: #616070;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  .value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  .buy {
+    margin-top: 16px;
+    height: 40px;
+    width: 160px;
+    background-color: $blue-color;
+    border-radius: 5px;
+    display: flex;
+    justify-content: center;
+
+    &__link {
+      padding: 12px 70px;
+      cursor: pointer;
+      color: #fff;
+      text-decoration: none;
+
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      text-transform: uppercase;
+    }
+
+    &__link:hover {
+      color: $blue-color;
+      background-color: #fff;
+      border-radius: 5px;
+      border: 1px solid $blue-color;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover